std/math/operations.d

```diff
@@ -1693,3 +1693,306 @@ if (isFloatingPoint!T)
         }
     }}
 }
+
+package(std): // not yet public
+
+struct FloatingPointBitpattern(T)
+if (isFloatingPoint!T)
+{
+    static if (T.mant_dig <= 64)
+    {
+        ulong mantissa;
+    }
+    else
+    {
+        ulong mantissa_lsb;
+        ulong mantissa_msb;
```

Comment:
So this PR would immediately break 128-bit real targets then?

Reply:
What do you mean with "break"? 128-bit reals are not supported by this code. If you call `FloatingPointBitpatten` with a `real` on such a target you'll get an `static assert` message.

Reply:
Ok, I had missed that this type only got instantiated inside a `mant_dig == 64` block. Just threw me off that you introduced these but do nothing in with them.